kyuubi-common/src/main/scala/org/apache/kyuubi/config/KyuubiConf.scala

```diff
@@ -657,6 +657,13 @@ object KyuubiConf {
       .checkValue(_ > 0, "the maximum must be positive integer.")
       .createWithDefault(10)
 
+  val SESSION_ENGINE_SYNC_INIT: ConfigEntry[Boolean] =
+    buildConf("session.engine.sync.init")
+      .doc("When opening kyuubi session, whether to init backend engine synchronously")
+      .version("1.4.0")
+      .booleanConf
+      .createWithDefault(true)
```

Comment:
Can we also add a `todo` here to make it `false` by default(maybe we shall rename the conf to xx.launch.async and set false now)